File: server/app/service/autoCode/te/casbin_rule/request/casbin_rule.go
package request

type CreateCasbinRule struct {
    PType string `p:"pType" v:"required|length:1,1000#请输入pType字段|pType字段长度为:min到:max位"`
    V0 string `p:"v0" v:"required|length:1,1000#请输入v0字段|v0字段长度为:min到:max位"`
    V1 string `p:"v1" v:"required|length:1,1000#请输入v1字段|v1字段长度为:min到:max位"`
    V2 string `p:"v2" v:"required|length:1,1000#请输入v2字段|v2字段长度为:min到:max位"`
    V3 string `p:"v3" v:"required|length:1,1000#请输入v3字段|v3字段长度为:min到:max位"`
    V4 string `p:"v4" v:"required|length:1,1000#请输入v4字段|v4字段长度为:min到:max位"`
    V5 string `p:"v5" v:"required|length:1,1000#请输入v5字段|v5字段长度为:min到:max位"`
}

type UpdateCasbinRule struct {
    PType string `p:"pType" v:"required|length:1,1000#请输入pType字段|pType字段长度为:min到:max位"`
    V0 string `p:"v0" v:"required|length:1,1000#请输入v0字段|v0字段长度为:min到:max位"`
    V1 string `p:"v1" v:"required|length:1,1000#请输入v1字段|v1字段长度为:min到:max位"`
    V2 string `p:"v2" v:"required|length:1,1000#请输入v2字段|v2字段长度为:min到:max位"`
    V3 string `p:"v3" v:"required|length:1,1000#请输入v3字段|v3字段长度为:min到:max位"`
    V4 string `p:"v4" v:"required|length:1,1000#请输入v4字段|v4字段长度为:min到:max位"`
    V5 string `p:"v5" v:"required|length:1,1000#请输入v5字段|v5字段长度为:min到:max位"`
}

type FindCasbinRule struct {
    PType string `p:"pType" v:"required|length:1,1000#请输入pType字段|pType字段长度为:min到:max位"`
    V0 string `p:"v0" v:"required|length:1,1000#请输入v0字段|v0字段长度为:min到:max位"`
    V1 string `p:"v1" v:"required|length:1,1000#请输入v1字段|v1字段长度为:min到:max位"`
    V2 string `p:"v2" v:"required|length:1,1000#请输入v2字段|v2字段长度为:min到:max位"`
    V3 string `p:"v3" v:"required|length:1,1000#请输入v3字段|v3字段长度为:min到:max位"`
    V4 string `p:"v4" v:"required|length:1,1000#请输入v4字段|v4字段长度为:min到:max位"`
    V5 string `p:"v5" v:"required|length:1,1000#请输入v5字段|v5字段长度为:min到:max位"`
}

type GetCasbinRuleList struct {
    PType string `p:"pType"`
    V0 string `p:"v0"`
    V1 string `p:"v1"`
    V2 string `p:"v2"`
    V3 string `p:"v3"`
    V4 string `p:"v4"`
    V5 string `p:"v5"`
	PageInfo
}

File: server/app/service/autoCode/fe/casbin_rule/api/casbin_rule.js
import service from '@/utils/request'

// @Tags CasbinRule
// @Summary 创建CasbinRule
// @Security ApiKeyAuth
// @accept application/json
// @Produce application/json
// @Param data body model.CasbinRule true "创建CasbinRule"
// @Success 200 {string} string "{"success":true,"data":{},"msg":"获取成功"}"
// @Router /casbinRule/createCasbinRule [post]
export const createCasbinRule = (data) => {
     return service({
         url: "/casbinRule/createCasbinRule",
         method: 'post',
         data
     })
 }


// @Tags CasbinRule
// @Summary 删除CasbinRule
// @Security ApiKeyAuth
// @accept application/json
// @Produce application/json
// @Param data body model.CasbinRule true "删除CasbinRule"
// @Success 200 {string} string "{"success":true,"data":{},"msg":"删除成功"}"
// @Router /casbinRule/deleteCasbinRule [delete]
 export const deleteCasbinRule = (data) => {
     return service({
         url: "/casbinRule/deleteCasbinRule",
         method: 'delete',
         data
     })
 }

// @Tags CasbinRule
// @Summary 删除CasbinRule
// @Security ApiKeyAuth
// @accept application/json
// @Produce application/json
// @Param data body request.IdsReq true "批量删除CasbinRule"
// @Success 200 {string} string "{"success":true,"data":{},"msg":"删除成功"}"
// @Router /casbinRule/deleteCasbinRule [delete]
 export const deleteCasbinRuleByIds = (data) => {
     return service({
         url: "/casbinRule/deleteCasbinRuleByIds",
         method: 'delete',
         data
     })
 }

// @Tags CasbinRule
// @Summary 更新CasbinRule
// @Security ApiKeyAuth
// @accept application/json
// @Produce application/json
// @Param data body model.CasbinRule true "更新CasbinRule"
// @Success 200 {string} string "{"success":true,"data":{},"msg":"更新成功"}"
// @Router /casbinRule/updateCasbinRule [put]
 export const updateCasbinRule = (data) => {
     return service({
         url: "/casbinRule/updateCasbinRule",
         method: 'put',
         data
     })
 }


// @Tags CasbinRule
// @Summary 用id查询CasbinRule
// @Security ApiKeyAuth
// @accept application/json
// @Produce application/json
// @Param data body model.CasbinRule true "用id查询CasbinRule"
// @Success 200 {string} string "{"success":true,"data":{},"msg":"查询成功"}"
// @Router /casbinRule/findCasbinRule [get]
 export const findCasbinRule = (params) => {
     return service({
         url: "/casbinRule/findCasbinRule",
         method: 'get',
         params
     })
 }


// @Tags CasbinRule
// @Summary 分页获取CasbinRule列表
// @Security ApiKeyAuth
// @accept application/json
// @Produce application/json
// @Param data body request.PageInfo true "分页获取CasbinRule列表"
// @Success 200 {string} string "{"success":true,"data":{},"msg":"获取成功"}"
// @Router /casbinRule/getCasbinRuleList [get]
 export const getCasbinRuleList = (params) => {
     return service({
         url: "/casbinRule/getCasbinRuleList",
         method: 'get',
         params
     })
 }